src/mode/modeHandler.ts

```diff
@@ -145,6 +146,7 @@ export class VimState {
    * by us or by a mouse action.
    */
   public whatILastSetTheSelectionTo: vscode.Selection;
+<<<<<<< b9a0b15742532ecc54e66ee04c417d82e70d29dc
```

Comment:
Be careful, you want to use `git add . && git rebase --continue`. Using a single & will cause a process to be backgrounded. 

But yeah, it's annoying and it eventually happens to everyone. No worries :)